### PR TITLE
Remove restriction when fetching blobs

### PR DESCRIPTION
**Pull request by lcarva**
The custom rego function `ec.oci.blob` limits downloads to 10MB. This is often used to fetch SBOMs which could be larger. This commit removes the restrictions altogether and instead relies on the constraints set on whichever environment is running EC.

Ref: KFLUXBUGS-1667

### CODE REVIEW

File: internal/rego/oci/oci.go [hunk 1]
```diff
@@ -177,8 +177,6 @@ func registerOCIImageFiles() {
 	rego.RegisterBuiltin2(&decl, ociImageFiles)
 }
 
-const maxBytes = 10 * 1024 * 1024 // 10 MB
```
Comment (nickboldt) [inline internal/rego/oci/oci.go:180]: or just bump this to 30 MB?

Comment (lcarva) [reply internal/rego/oci/oci.go:180]: That seem arbitrary as well.

I wonder if making this configurable, e.g. via env var, is a better approach.

Comment (simonbaird) [reply internal/rego/oci/oci.go:180]: What was the thinking behind adding a limit initially? Maybe there's still some value in it, even if it's bumped to 100MB or something.

**Review by simonbaird — APPROVED** [review]

**Review by zregvart — APPROVED** [review]

File: internal/rego/oci/oci.go (hunk 1 above)
Comment (lcarva) [reply internal/rego/oci/oci.go:180]: Let's try out without any restrictions. We can always go back later on if we find a good reason.

**Pull request merged**